openrtb-core/src/main/protobuf/openrtb.proto

```diff
@@ -758,6 +762,10 @@ message BidRequest {
     // For example, "foo.com".
     optional string domain = 3;
 
+    // The taxonomy in use.
+    // Refer to the enum com.iabtechlab.adcom.v1.enums.CategoryTaxonomy for values.
+    optional int32 cattax = 16 [default = 1];
```

Comment:
`Site.cattax` has missed `default 1` note in https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md

Reply:
Added https://github.com/InteractiveAdvertisingBureau/openrtb2.x/pull/74.